tests/utils/test_github_sponsors.py

```diff
@@ -41,3 +54,24 @@ def test_parse_gh_org_funding_json(self):
         self.assertEqual(
             expected_dict, gs.parse_gh_org_funding_json(json.loads(example_json))
         )
+
+    def test_get_gh_top_contributors_json(self):
+        # note: this could change one day! if so, will be
+        # a happy thing :)
+        usernames_in_top_contribs = ["jmelot", "jspeed-meyers"]
+        got = gs.get_gh_top_contributors_json("georgetown-cset/funder-finder")
+        for username in usernames_in_top_contribs:
+            self.assertEqual(username in got, True)
+
+    def test_get_gh_user_gh_sponsors(self):
+        gs.get_gh_user_gh_sponsors("ljharb")
```

Comment:
Are you checking whether this raises an exception?

Reply:
I'm not sure what I'm doing :) My intention was to exercise this function without specifying what the result should be since it could change.

I was hoping this minimal test would fail if future code dorked up that `get_gh_user_gh_sponsors` function.

Do you think this block should use a
```python
try:

except:
    raise
```

construct?

Reply:
Ah I see... no, I think it's ok as-is. Thanks for explaining!